lib/elixir/lib/process.ex

```diff
@@ -208,6 +208,7 @@ defmodule Process do
       end
 
   """
+  @spec sleep(timeout) :: :ok | no_return
```

Comment:
I thought it was OK to use `no_return` when a process is continuously running, as in the case of timeout = `:infinity`